### ISSUE TITLE
is this discontinued? 

### ISSUE THREAD

**Issue by Stooovie** [koying/jellyfin_ha#43]
the default Jellyfin integ in HA is lacking, this one looks more capable, but I can't get it to work in HA 2023.5.x. Are there any updates planned? Thanks!

**Comment by andreasc1** [comment]
same feeling i have, i recently moved from plex to jellyfin and the official integration is lacking functionality, this one seems better but it is clashing with the official somehow..

**Comment by Stooovie** [comment]
Official is uninstalled and yet this cannot be set up in 2023.8 (and earlier) at all. "Invalid flow specified"

**Comment by PixelPusher247** [comment]
With the update to Jellyfin 10.9 the official integration is not working anymore and it doesn't look like my issue is getting any attention, what is the state on this integration, is there any development here planned?

**Comment by formatBCE** [comment]
Guess this one is dead.

Official integration isn't really helpful, and creates abandoned media_player entities here and there....

I'm curious if MQTT can help. There's Websocket plugin for Jellyfin, that can post info into MQTT. It would be cool, as this would be push instead of poll.

**Comment by PixelPusher247** [comment]
> Guess this one is dead.
> 
> Official integration isn't really helpful, and creates abandoned media_player entities here and there....
> 
> I'm curious if MQTT can help. There's Websocket plugin for Jellyfin, that can post info into MQTT. It would be cool, as this would be push instead of poll.

Yeah, that's what I ended up doing, using the Webhook plugin in Jellyfin I post messages to MQTT when playback starts, pauses or stops, I only use this for the Jellyfin client on my Nvidia Shield so it wasn't too much of a hassle to set up and works instantly, even faster than with the official integration when it was working.
